metrics/grafana/ticdc.json

```diff
@@ -1129,6 +1129,82 @@
           "transform": "timeseries_aggregations",
           "type": "table"
         },
+        {
+          "columns": [
+            {
+              "$$hashKey": "object:139",
```

Comment:
```suggestion
              "$$hashKey": "object:140",
```

Reply:
id 139 is conflicted with the metric of `Old value seek duration`, which is introduced in another PR today